### PR TITLE
add evaluate.py to test some real use cases

### PR DESCRIPTION
**Pull request by romaissaMe**
this is far from being the final code I was just testing if the logic works then I couldn't continu because of trying to fix the errors

### CODE REVIEW

**Comment by romaissaMe** [comment]
@achab

File: app/api/routers/ask_gpt.py [hunk 1]
```diff
@@ -5,18 +5,27 @@
...
-openai.api_key=os.getenv("OPENAI_API_KEY")
+ask_router = APIRouter()
+openai.api_key = os.getenv("OPENAI_API_KEY")
+
 
 @ask_router.get('/ask')
-async def ask(question:str,text:str|None):
-    anonymized_text= anonymize_text(text)
-    resp=openai.ChatCompletion.create(
+async def ask(question: str, text: str | None):
```
Comment (achab) [inline app/api/routers/ask_gpt.py:13]: I can see that you are using a linter to vscode or pycharm :)

File: use_cases/payslip/evaluate.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    with open('results.txt', "a") as res_file:
+        for question in questions:
+            nq += 1
+            payload = {'question': question, 'text': text}
+            res = req.get('http://127.0.0.1:8000/ask', params=payload)
+            res_file.write(res.json()['response'].replace('\n', ' ') + '\n')
+            if nq != 3:
+                time.sleep(1)
+            else:
+                time.sleep(60)
```
Comment (achab) [inline use_cases/payslip/evaluate.py:27]: nice retry mechanism

File: use_cases/payslip/evaluate.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+
+
```
Comment (achab) [inline use_cases/payslip/evaluate.py:30]: useless lines

**Review by achab — APPROVED** [review]

**Comment by achab** [comment]
Good job, feel free to merge your work!

File: app/api/routers/ask_gpt.py (hunk 1 above)
Comment (romaissaMe) [reply app/api/routers/ask_gpt.py:13]: always very sharp remarks! xd

**Pull request merged**